sdcm/cluster.py

```diff
@@ -2084,6 +2084,8 @@ def web_install_scylla(self, scylla_version: Optional[str] = None) -> None:
 
         if match := re.match(r'\D*(\d+\.\d+)', scylla_version or git_branch):
             version = f"nightly-{match.group(1)}"
+        elif git_branch.split('/')[1] in ['enterprise', 'master']:
+            version = f"nightly-{git_branch.split('/')[1]}"
```

Comment:
```suggestion
        elif '/' in git_branch and git_branch.split('/')[1] in ['enterprise', 'master']:
            version = f"nightly-{git_branch.split('/')[1]}"
```

Reply:
Also, there is no `enterprise` branch in the sct, not quite sure what it is related to

Reply:
right, i noticed, all of this is not a good practice. well then how about just setting 'master' explicitly as the default?

Reply:
If enterprise or master, use master.

Reply:
It would install the wrong thing, if we choose master, it would install nightly-master.

If we want to cover nightly-enterprise, we need to distinguish it somehow.

Making the logic here based on SCT branch name is a bit fragile.

If we need a parameter to be passed down by scylla-pkg, let's add it.

Meanwhile I think they are passing down scylla_repo, which doesn't exactly help us.

Reply:
we don't really need to test enterprise branch for web install.
(In theory also master isn't that required...)

Reply:
ok, updated it to be left with `master` only